### PR TITLE
Configure nightly builds on Jenkins

### PR DESCRIPTION
**Pull request by jamiesnape**

### CODE REVIEW

**Comment by stonier** [comment]
Just for my future reference, the 'H' here is:

```
To allow periodically scheduled tasks to produce even load on the system, the symbol H (for “hash”) should be used wherever possible.

For example, using 0 0 * * * for a dozen daily jobs will cause a large spike at midnight. In contrast, using H H * * * would still execute each job once a day, but not all at the same time, better using limited resources.
```

File: Jenkinsfile [hunk 1]
```diff
@@ -32,6 +32,13 @@
  *****************************************************************************/
 
 node('linux_xenial_unprovisioned') {
+  def triggers = []
+  if (env.BRANCH_NAME == 'master') {
+    triggers << cron('H H(5-6) * * *')
```
Comment (stonier) [inline Jenkinsfile]: @jamiesnape why not just `H H * * *` or is the evening of the load not quite that smart?

Comment (jamiesnape) [reply Jenkinsfile]: It needs to occur later than the nightly packages are built for Drake.

Comment (stonier) [reply Jenkinsfile]: Got it.

**Pull request merged**